### ISSUE TITLE
GRSciColl - add identifier type CITES

### ISSUE THREAD

**Issue by ManonGros** [gbif/registry#405]
A number of scientific institutions are registered on CITES: https://cites.org/eng/common/reg/e_si.html
In order to improve database interoperability and because species trade is a serious issue in conservation, I would like to be able to add CITES identifiers to GRSciColl entries.

@marcos-lg could we add a "CITE" identifier type?

**Comment by marcos-lg** [comment]
@ManonGros do we need to validate these identifiers according to the rules [here](https://cites.org/eng/common/reg/e_si.html)?

**Comment by MattBlissett** [comment]
> could we add a "CITE" identifier type?

I think that's a typing error, as the type would be `CITES` (**C**onvention on **I**nternational **T**rade in **E**ndangered **S**pecies [of Wild Fauna and Flora]).

**Comment by ManonGros** [comment]
Yes thanks Matt! you are right, I corrected it in the original post.
